setup.py

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env python
```

Comment:
setup.py can be deleted, we don't need it anymore as we don't use setuptools.

Reply:
Good point! `setup.py` and `setup.cfg` are removed now.